src/util/util_lru.h

```diff
@@ -0,0 +1,69 @@
+#include <cstdint>
+#include <list>
+#include <unordered_map>
+
+namespace dxvk {
+
+  template<typename T>
+  class lru_list {
+
+  public:
+    typedef typename std::list<T>::const_iterator lru_iterator;
```

Comment:
why not just call this `const_iterator`? Might as well make this work with C++ forach thingy.